### DISCUSSION TITLE
The server is temporarily unable to service your request due to maintenance downtime or capacity problems

### DISCUSSION THREAD

**Discussion by Bojack84** [aaronwmorris/indi-allsky#1245] [Q&A]
I am pretty sure that I am having this issue as it says that usage of /: is at 95.1% of 99 GB.  I think I need to delete some videos or pictures but I want to make sure before I do this.  This is running on a VM on proxmox.  It has been pretty flawless but all of a sudden I got this error.  

![image](https://github.com/aaronwmorris/indi-allsky/assets/23743816/527efb3b-730b-4a7b-ba13-8d7fc904b548)

![image](https://github.com/aaronwmorris/indi-allsky/assets/23743816/5ff7a269-c1a9-4716-8bb9-3e9e72f56ee5)

Please let me know if you need anything else from me.

**Comment by aaronwmorris** [comment]
You probably need to check the logs at `/var/log/indi-allsky/webapp-indi-allsky.log`.

It sounds like the gunicorn process needs to be restarted.

```
systemctl --user restart gunicorn-indi-allsky
```

**Reply by Bojack84** [reply]
When I try to check the log it is empty and I get an error when I try that command. I even tried the command with sudo in front of it. 

![image](https://github.com/aaronwmorris/indi-allsky/assets/23743816/2877437b-4fe5-47a6-911d-af9ca6f56d8c)

![image](https://github.com/aaronwmorris/indi-allsky/assets/23743816/a1b2a8dd-5606-4fb3-8bbf-a6807cb892b2)

**Comment by aaronwmorris** [comment]
It's possible you ran out of disk space and left the system in a weird state.  I would free up some space and just fully reboot.

If that does not work, re-run `setup.sh` and see if that will fix any permissions.

**Reply by Bojack84** [reply]
I agree that it is fully out of space but I searched and couldnt find out how to clear up space on it.  Where should I navigate to delete some videos/pictures?

**Comment by aaronwmorris** [comment]
Merged #1247 with a new script to expire images and videos from the CLI.  You may specify more aggressive cutoffs than what is configured in indi-allsky.

```
source virtualenv/indi-allsky/bin/activate

# expire images older than 15 days
./misc/expire_images.py --days 15
```

**Reply by Bojack84** [reply]
I am not sure how to use what you put in here.  I was wondering where I can go to delete some files like older videos/images.  Can you point me in that direction?

**Reply by aaronwmorris** [reply]
The images and videos will be under the `/var/www/html/allsky/images/` folder.  There will be a sub folder for the camera named something like `ccd_xxxxxx-xxxx-xxx-xxx-xxxxxxxxxxx`.

I just made a major change to the folder layouts right after you opened this issue, so there might be some variation.  You may see folders by date, which you can delete the oldest folders in that case.  The new style will have "exposures", "panoramas", etc, and each of these will have dated folders.

**Reply by Bojack84** [reply]
Thank you for that location.  It helped me clear it all out and now it is back up and running.  I did go in and set the data storage limits.
